<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectIngredients</class>
 <widget class="QMainWindow" name="selectIngredients">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Recipe name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="recipeName"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="ingredientList"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Amount</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="amountLine"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameLine"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="typeBox">
        <item>
         <property name="text">
          <string>Mg</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>G</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Kg</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ml</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Liters</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Spoons</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tea spoons</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Caps</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addIngredientButton">
        <property name="text">
         <string>Add ingredient</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="finishButton">
        <property name="text">
         <string>Srat adding steps</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
